test/nightly/azureResourceCreateAccount.test.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { CosmosDBManagementModels } from 'azure-arm-cosmosdb';
+import * as vscode from 'vscode';
+import { randomUtils } from '../../extension.bundle';
+import { longRunningTestsEnabled, testUserInput } from '../global.test';
+import { accountList, api, client, resourceGrouList } from './global.resource.test';
+
+suite('Create account', async function (this: Mocha.Suite): Promise<void> {
+    this.timeout(20 * 60 * 1000);
+
+    suiteSetup(async function (this: Mocha.Context): Promise<void> {
+        if (!longRunningTestsEnabled) {
+            this.skip();
+        }
+        this.timeout(15 * 60 * 1000);
+        await Promise.all([delayCreateAccount(5, /graph/), delayCreateAccount(10, /MongoDB/), delayCreateAccount(15, /SQL/)]);
```

Comment:
How are these numbers being selected?  5, 10, 15?  Why are they all different?  Are those average times?

Reply:
We can't create several accounts in the same time, so wait 5s to create the next account.